wolt_services.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from constants import WOLT_URL
+
+
+def check_if_restaurant_is_open(
+    restaurant_id: str, lat: float, lon: float
+) -> bool | str:
```

Comment:
Why are you returning a bool or a string? It's a bad practice.
As defined "check_if_restaurant_is_open", it a typical boolean check (is it actually open or not?).

In the case of not being able to retrive the status of the restaurant, raise an exception and handle it outside of the function.